README.md

```diff
@@ -72,116 +72,122 @@ For guidance on developing for PowerToys, please read the [developer docs](/doc/
+- Fixed signing of the setup custom actions dll in the new pipeline.
+- The Visual C++ redistributable was updated to 14.32.31332 and fixed an installer error when a newer version was installed. Thanks [@snickler](https://github.com/snickler)!
+- Updated the .NET dependency to 6.0.6.
 
 ### Development
 
-- New action added to GitHub to publish the winget package to PowerToys.
-- New action added to GitHub to publish for the Microsoft Store. Thanks [azchohfi](https://github.com/azchohfi)!
-- Documentation for installing the Windows App SDK dependencies and building the installer was updated.
-- FxCop removed from the PowerToys Run TimeZone plugin and was replaced with NetAnalyzers. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Clean up of the CA1031 warning suppression. Thanks [@davidegiacometti](https://github.com/davidegiacometti)!
+- Support for ARM64 binaries was added to the Microsoft Store submission task. Thanks [@azchohfi](https://github.com/azchohfi)!
+- Added code for a tool to help identify monitor IDs.
+- Support for ARM64 binaries was added to the winget package creation task.
+- Updated the Pull Request template to better reflect project changes.
+- Component Governance checks were re-activated on the new main branch.
+- CI is failing to run tests calling the newer WebView 2 version, so these were disabled until a fix is found.
+- Updated the tests SDK to 17.2.0.
+- Nuget package versions used in the solution were consolidated.
+- The CodeQL CI task was disabled in the repo, but was causing issues on forks, so it was removed.
+- A specific Newtonsoft.Json version was specified in tests to avoid a vulnerability present in previous versions.
+- FabricBot configurations were added to the repository.
+- Added a dependabot configuration for updating GitHub actions dependencies. Thanks [@naveensrinivasan](https://github.com/naveensrinivasan)!
+- Updated the check-spelling action and added quality of life fixes to the workflow. Thanks [@jsoref](https://github.com/jsoref)!
 
 #### Community contributions
 
 We'd like to directly mention certain contributors (in alphabetical order) for their continued community support this month and helping directly make PowerToys a better piece of software.  
 [@Aaron-Junker](https://github.com/Aaron-Junker), [@CleanCodeDeveloper](https://github.com/CleanCodeDeveloper), [@davidegiacometti](https://github.com/davidegiacometti), [@dend](https://github.com/dend), [@franky920920](https://github.com/franky920920), [@htcfreek](https://github.com/htcfreek), [@jay-o-way](https://github.com/jay-o-way), [@jsoref](https://github.com/jsoref), [@niels9001](https://github.com/niels9001), [@snickler](https://github.com/snickler).
 
-#### What is being planned for v0.60
+#### What is being planned for v0.61
 
-For [v0.60][github-next-release-work], we'll work on below:
+For [v0.61][github-next-release-work], we'll work on below:
 
```

Comment:
Somebody recently made a pr for a QR scanner

Reply:
It's a OCR tool. Not QR

Reply:
This is more of "what the core team is working on".

Reply:
> This is more of "what the core team is working on".

Does that really matter for users?